python-package/lightgbm/dask.py

```diff
@@ -467,7 +520,7 @@ def predict(self, X, **kwargs):
 
     predict.__doc__ = LGBMClassifier.predict.__doc__
 
-    def predict_proba(self, X, **kwargs):
+    def predict_proba(self, X: _DaskCollection, **kwargs: Any) -> _DaskCollection:
```

Comment:
Same about inability to pass `dask Series` as above.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3866/commits/8b19e04b05f83a38f5213c1103040f49d0a20988